UnityProject/Assets/Scripts/ScriptableObjects/Research/DefaultTechwebData.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Systems.Research;
+using NaughtyAttributes;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace ScriptableObjects.Research
+{
+	[CreateAssetMenu(fileName = "DefaultTechWebData", menuName = "ScriptableObjects/Systems/Techweb/DefaultTechwebData")]
+	public class DefaultTechwebData : ScriptableObject
```

Comment:
I'm comparing with the UML diagram and I don't see ``TechWebData`` nor ``Technology`` object

Reply:
`TechWebData` is the name that lets us distinct json data from in-game data.
`Technology` is the old name that bod used from looong ago, though I'm not entirely sure if the name "item" is good for this to be honest. I'll ask Cross about it.

Reply:
yeah the idea is implementing the UML architecture and following it as close as possible.